core/src/main/java/org/keycloak/representations/account/LinkedAccountsRepresentation.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.representations.account;
+
+import java.util.Set;
+
+/**
+ *
+ * @author Stan Silvert
+ */
+public class LinkedAccountsRepresentation {
+    private Set<LinkedAccountRepresentation> linkedAccounts;
+    private boolean hasAlternativeAuthentication;
```

Comment:
That means making two REST calls instead of one, but I'm fine with it.  Agree that hasAlternativeAutentication doesn't really seem like it belongs in this class.